updater/dart_cli/.gitignore

```diff
@@ -1,3 +1,4 @@
 # https://dart.dev/guides/libraries/private-files
 # Created by `dart pub`
 .dart_tool/
+pubspec.lock
```

Comment:
Minor but can we also `git rm --cached updater/dart_cli/pubspec.lock`?

Reply:
Yes, I thought I had.  🤔

Reply:
weird...maybe you forgot to commit the changes? I would've expected to see the deleted lockfile as part of this diff 🤔